apis/repositories/order_repository.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from typing import List, Dict, Any
+from datetime import datetime, timedelta
+
+class OrderRepository:
+    @classmethod
+    def get_pending_orders(cls):
+        return Order.objects.filter(status='pending')
+    
+    @classmethod
+    def update_order_status(cls, order_id: int, status: str) -> None:
+        Order.objects.filter(id=order_id).update(
+            status=status,
+            updated_at=datetime.now()
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using timezone-aware datetime function</b></div>
<div id="fix">

Consider using `timezone.now()` instead of `datetime.now()` to ensure timezone-aware timestamps in Django
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,4 +1,5 @@
 from django.db import models
 from typing import List, Dict, Any
 from datetime import timedelta
 +from django.utils import timezone

 @@ -14,7 +14,7 @@
 -            updated_at=datetime.now()
 +            updated_at=timezone.now()

 @@ -28,7 +28,7 @@
 -            updated_at=datetime.now()
 +            updated_at=timezone.now()

 @@ -34,7 +34,7 @@
 -            created_at__lt=datetime.now() - timedelta(days=days),
 +            created_at__lt=timezone.now() - timedelta(days=days),
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/Django/pull/3#issuecomment-2523273297>#5de351</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged